packages/contracts/hardhat/deployments/ApeLendingStrategy.deployment.ts

```diff
@@ -0,0 +1,81 @@
+import { DeployConfig, BaseDeploymentService, BaseInitArgs, wrap } from '@eonian/upgradeable'
+import { BlockchainType } from "../../hardhat.config";
+import { DeployFunction, DeployResult, Deployment } from '@eonian/hardhat-deploy/types';
+
+const HOUR = 60 * 60; // hour in seconds
+
+export interface ApeSwapLendingStrategyDeploymentOptions {
+    /** Name of asset */
+    asset: string;
+}
+
+export interface ApeSwapLendingStrategyDeployConfig extends DeployConfig {
+    options: ApeSwapLendingStrategyDeploymentOptions;
+}
+
+/**
+ * Base config for ApeSwap Lending strategy contract
+ */
+export const base: Omit<DeployConfig, 'tags' | 'dependencies'> = {
+    contract: "ApeLendingStrategy",
+    // Not possible to deploy on testnet, have wide range of thrid party protocols-dependencies
+    chains: [
+        BlockchainType.Mainnet,
+        BlockchainType.Local,
+    ],
+}
+
+export function generateApeSwapLendingStrategyDeployment(options: ApeSwapLendingStrategyDeploymentOptions): DeployFunction {
+    const config: ApeSwapLendingStrategyDeployConfig = {
+        ...base,
+        options,
+        dependencies: [`Vault|Asset[${options.asset}]`],
```

Comment:
why `|` and not more readable `_`?

Reply:
Because we can have a case when we use `_` in the contract or tag name. `|` is a clear separator, it is more simular to `/` than `_`